### PR TITLE
Disallow iFrame embeds

### PR DESCRIPTION
**Pull request by berndverst**
Disallow iFrame embeds

### CODE REVIEW

**Review by artursouza — APPROVED** [review]

File: daprdocs/config.toml [hunk 1]
```diff
@@ -10,6 +10,12 @@ enableGitInfo = true
 # Language Configuration
 languageCode = "en-us"
 
+[server]
+[[server.headers]]
+  for = '/**.html'
```
Comment (pkedy) [inline daprdocs/config.toml:15]: In my local testing, `/**.html` did not work. I had to use `/**` for the response headers to return. However, when I did that, Chrome developer extensions showed something interesting for `.js` files.  I'm not sure its its a real issue or not.

image

Comment (pkedy) [reply daprdocs/config.toml:15]: My config snippet was:

```toml
[server]
[[server.headers]]
  for = '/**'
  [server.headers.values]
    Content-Security-Policy = 'script-src localhost:1313'
    Referrer-Policy = 'strict-origin-when-cross-origin'
    X-Content-Type-Options = 'nosniff'
    X-Frame-Options = 'DENY'
    X-XSS-Protection = '1; mode=block'
```

**Comment by AaronCrawfis** [comment]
[server] works for local dev only. For production we'll need to define a Static Web Apps configuration file and update the global headers: https://docs.microsoft.com/en-us/azure/static-web-apps/configuration#global-headers

**Comment by berndverst** [comment]
See #1820 @pkedy  @AaronCrawfis

**Comment by berndverst** [comment]
I'm closing this because it only applies to local development and we don't really need this protection there.

**Pull request closed without merging**